### PR TITLE
Fix pts wrap

### PR DESCRIPTION
**Pull request by elv-reza**
issue: https://github.com/eluv-io/avpipe/issues/44

### CODE REVIEW

File: libavpipe/src/avpipe_xc.c [hunk 1]
```diff
@@ -3623,6 +3606,12 @@ avpipe_xc(
             break;
         }
 
+        if (input_packet->flags & AV_PKT_FLAG_CORRUPT) {
+            elv_warn("packet corrupt pts=%"PRId64, input_packet->pts);
+            av_packet_free(&input_packet);
+            continue;
+        }
```
Comment (elv-serban) [inline libavpipe/src/avpipe_xc.c:3613]: I would prefer to put this in a separate PR - or add a flag to enable/disable.

**Review by elv-serban — APPROVED** [review]
Looks good!
Though I would prefer we either ifdef or add a flag for the AV_PKG_FLAG_CORRUPT so we can test both with and without this logic.

File: libavpipe/src/avpipe_xc.c (hunk 1 above)
Comment (elv-reza) [reply libavpipe/src/avpipe_xc.c:3613]: FFmpeg does this all the time, I am not sure why you want to disable it. BTW, we can add it later
